CHANGELOG.md

```diff
@@ -1094,6 +1094,8 @@ activity and on its child transactions (which can be actuals, refunds, and adjus
 
 ## [unreleased]
 
+- Refactor the importers to use a unified approach for all entities being imported
```

Comment:
`importers (including uploads controllers)`?

Reply:
I tend to err on the side of having a changelog entry rather than not, to make things easier to trace across the history of the project. The checklist suggests "unless it's a small tweak" with no impact outside the dev team, so I tend to think that big tweaks should be logged, even if they have no impact outside the dev team.